### DISCUSSION TITLE
Can this crate be used to create a regular old QT Widgets app?

### DISCUSSION THREAD

**Discussion by edisno** [woboq/qmetaobject-rs#245] [General]
It seems qt_core, qt_widgets etc is unmaintained, can this crate be used instead with a different approach?  If not, what is missing?

**Comment by ogoffart** [comment]
No, this crate is only for QML

The Qt Widgets API has a much bigger API surface than QML and it would be a much bigger work to expo it with a nice Rust API

**Comment by ratijas** [comment]
If you are asking that question, then it's definitely not for you :)

Technically you could expose list/table/item models and export dylibs as plugins for your Widgets based app. But writing Qt Widgets app from scratch in Rust would be pure madness, probably not worth it.
